server/src/server/app.ts

```diff
@@ -1,17 +1,19 @@
+import "dotenv/config";
 import express from "express";
 import bodyParser from "body-parser";
 import cors from "cors";
 import { initializeApplication } from "./server";
+const { photosRouter } = require("./routes/photos");
 
 export const app = express();
 
 app.use(bodyParser.json());
-
 app.use(cors({
     origin: '*',
     credentials: true,
     optionsSuccessStatus: 200
 }));
+app.use('/photos', photosRouter);
```

Comment:
When you do merge conflicts, add `UNSPLASH_API_URL` and `UNSPLASH_API_KEY` to `environment.d.ts`